### ISSUE TITLE
Return valid response for unsupported mime-type 

### ISSUE THREAD

**Issue by miyconst** [Starcounter/Starcounter#2648]
Pages hang if try to access with header:
```json
"Accept": "text/xml"
```

The administrator console logs an exception:
```
System.Exception: Unsupported mime-type Other in request Accept header.
```

It should not only log the exception, but also return a valid http result to client.

**Comment by alemoi** [comment]
I will check this one! Thanks for reporting!

**Comment by miyconst** [comment]
Ping @alemoi.

**Comment by alemoi** [comment]
I was discussing this around and we decided that a strict mime type check is fine. So in this case a `406 Not acceptable` will be returned without logging the exception. What do you think about this?

**Comment by miyconst** [comment]
Looks like a desired behaviour, the client should know that the server works, but does not accept this mime type. And yes, this is not an error.

**Comment by alemoi** [comment]
Thank you for reporting! Fixed via 4be20c5588e8c340758bbc5c3bffa24db05f2128
